tensorflow/compiler/xla/service/gpu/gpu_layout_assignment.cc

```diff
@@ -58,8 +58,13 @@ HeuristicLayoutAssignment(const HloInstruction* instr,
                       DataLayout::kBatchYXDepth);
 
   // If we're not Volta or not fp16, the decision is easy: Use NCHW.
+#ifdef TENSORFLOW_USE_ROCM
+
+  if (!(instr->operand(0)->shape().element_type() == xla::PrimitiveType::F16)) {
+#elif
```

Comment:
it's unclear why this check should be changed. `IsVoltaOrLater` should always evaluate to `false` on ROCm. do we want `kAllNCHW` be always taken?

Reply:
I need to revisit ..might be IsVoltaOrLater was pulling in new dependencies. Let me confirm